fastq_filtrator.py

```diff
@@ -0,0 +1,103 @@
+## Functions
+# This is a function that returns "Pass" if gc content of a read is between gc_bounds and "Fail" if it's not
+def gc_filter(seq, gc_bounds):
+    if len(gc_bounds) == 2:
+        min_gc, max_gc = int(gc_bounds[0]), int(gc_bounds[1])
+    elif len(gc_bounds) == 1:
+        min_gc, max_gc = int(gc_bounds[0]), 0 
```

Comment:
Если одно число, то граница должна была быть верхней
```suggestion
        min_gc, max_gc = 0, int(gc_bounds[0])
```